lib/instrumentation/aws-sdk/v3/bedrock.js

```diff
@@ -55,7 +55,18 @@ function isStreamingEnabled({ commandName, config }) {
  */
 function recordEvent({ agent, type, msg }) {
   msg.serialize()
-  agent.customEventAggregator.add([{ type, timestamp: Date.now() }, msg])
+  const context = agent._contextManager.getContext()
+  const llmContext = Object.keys(context || {}).reduce((result, key) => {
+    if (key.indexOf('llm.') === 0) {
+      result[key] = context[key]
+    }
+    return result
+  }, {})
+
+  agent.customEventAggregator.add([
+    { type, timestamp: Date.now() },
+    Object.assign(msg, llmContext || {})
```

Comment:
This will update the `msg` object at the call site. Are you sure it shouldn't be `Object.assign({}, msg, llmContext || {})`?

Reply:
Would it matter? In original code we attach use msg object. Here we attach extra stuff to that msg.

Reply:
I don't know if it would matter. I'm assuming `msg` comes from the instrumented application code. In my opinion, we shouldn't update objects we don't own. Is that happening here?